src/Sulu/Bundle/ContentBundle/Resources/public/js/components/webspace/settings/analytics/main.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define([], function() {
+
+    'use strict';
+
+    var defaults = {};
+
+    return {
+
+        defaults: defaults,
```

Comment:
this component is only bootstraped and will be filled in one of the following PRs